code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -3572,11 +3571,31 @@ datum
+												"Aug 01" = 86, "Aug 02" = 79, "Aug 03" = 71, "Aug 04" = 64, "Aug 05" = 57, "Aug 06" = 50, "Aug 07" = 43, "Aug 08" = 36, "Aug 09" = 29, "Aug 10" = 21, "Aug 11" = 14, "Aug 12" = 7, "Aug 13" = 0, "Aug 14" = 6, "Aug 15" = 12, "Aug 16" = 19, "Aug 17" = 25, "Aug 18" = 31, "Aug 19" = 38, "Aug 20" = 44, "Aug 21" = 50, "Aug 22" = 56, "Aug 23" = 62, "Aug 24" = 69, "Aug 25" = 75, "Aug 26" = 81, "Aug 27" = 88, "Aug 28" = 94, "Aug 29" = 100, "Aug 30" = 93, "Aug 31" = 86,
+												"Sep 01" = 79, "Sep 02" = 71, "Sep 03" = 64, "Sep 04" = 57, "Sep 05" = 50, "Sep 06" = 43, "Sep 07" = 36, "Sep 08" = 29, "Sep 09" = 21, "Sep 10" = 14, "Sep 11" = 7, "Sep 12" = 0, "Sep 13" = 7, "Sep 14" = 13, "Sep 15" = 20, "Sep 16" = 27, "Sep 17" = 33, "Sep 18" = 40, "Sep 19" = 47, "Sep 20" = 53, "Sep 21" = 60, "Sep 22" = 67, "Sep 23" = 73, "Sep 24" = 80, "Sep 25" = 87, "Sep 26" = 93, "Sep 27" = 100, "Sep 28" = 93, "Sep 29" = 86, "Sep 30" = 79,
+												"Oct 01" = 71, "Oct 02" = 64, "Oct 03" = 57, "Oct 04" = 50, "Oct 05" = 43, "Oct 06" = 36, "Oct 07" = 29, "Oct 08" = 21, "Oct 09" = 14, "Oct 10" = 7, "Oct 11" = 0, "Oct 12" = 6, "Oct 13" = 12, "Oct 14" = 19, "Oct 15" = 25, "Oct 16" = 31, "Oct 17" = 38, "Oct 18" = 44, "Oct 19" = 50, "Oct 20" = 56, "Oct 21" = 62, "Oct 22" = 69, "Oct 23" = 75, "Oct 24" = 81, "Oct 25" = 88, "Oct 26" = 94, "Oct 27" = 100, "Oct 28" = 93, "Oct 29" = 86, "Oct 30" = 79, "Oct 31" = 71,
+												"Nov 01" = 64, "Nov 02" = 57, "Nov 03" = 50, "Nov 04" = 43, "Nov 05" = 36, "Nov 06" = 29, "Nov 07" = 21, "Nov 08" = 14, "Nov 09" = 7, "Nov 10" = 0, "Nov 11" = 7, "Nov 12" = 13, "Nov 13" = 20, "Nov 14" = 27, "Nov 15" = 33, "Nov 16" = 40, "Nov 17" = 47, "Nov 18" = 53, "Nov 19" = 60, "Nov 20" = 67, "Nov 21" = 73, "Nov 22" = 80, "Nov 23" = 87, "Nov 24" = 93, "Nov 25" = 100, "Nov 26" = 93, "Nov 27" = 86, "Nov 28" = 79, "Nov 29" = 71, "Nov 30" = 64,
+												"Dec 01" = 57, "Dec 02" = 50, "Dec 03" = 43, "Dec 04" = 36, "Dec 05" = 29, "Dec 06" = 21, "Dec 07" = 14, "Dec 08" = 7, "Dec 09" = 0, "Dec 10" = 6, "Dec 11" = 12, "Dec 12" = 19, "Dec 13" = 25, "Dec 14" = 31, "Dec 15" = 38, "Dec 16" = 44, "Dec 17" = 50, "Dec 18" = 56, "Dec 19" = 62, "Dec 20" = 69, "Dec 21" = 75, "Dec 22" = 81, "Dec 23" = 88, "Dec 24" = 94, "Dec 25" = 100, "Dec 26" = 93, "Dec 27" = 86, "Dec 28" = 79, "Dec 29" = 71, "Dec 30" = 64, "Dec 31" = 57)
 
-			does_react(var/datum/reagents/holder)
-				return time2text(world.realtime, "MMM DD") in full_moon_days_2053 //just doesn't react unless it's a full moon
+			on_reaction(var/datum/reagents/holder, var/created_volume)
+				var/my_atom = holder.my_atom
+				if(!my_atom) return
+				var/turf/reaction_loc = get_turf(my_atom)
+				if(!reaction_loc) return
+				var/date = time2text(world.realtime, "MMM DD")
+				if(prob((moon_list_2053[date] ** 4) / 100))
```

Comment:
I do not think this expression is correct. Check your math.

Reply:
Must have been really tired when I wrote that. Fixed.